### DISCUSSION TITLE
Show created / participated discussions per repository

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#5504] [Discussions]
If you head to a repository and click on `Discussions`, one can't easily find created or participated discussions.

Above / Below the `Categories` section could be a new `Personal` section which shows created and participated (or probably subscribed) discussion links.
With this addition, it's easier to continue discussing.

### Example
![image](https://user-images.githubusercontent.com/20150243/132315224-a3500f10-519f-4c39-a566-4893ca4e943a.png)

**Comment by laymonage** [comment]
This is currently possible using the `author:@me` or `involves:@me` queries.

**Reply by Zerotask** [reply]
Thanks!

Then there are just the 2 menu items missing.
